src/apps/ENGINE/file_service.cpp

```diff
@@ -277,21 +281,25 @@ BOOL FILE_SERVICE::LoadFile(const char *file_name, char **ppBuffer, uint32_t *dw
     if (ppBuffer == nullptr)
         return false;
 
-    auto *const hFile = _CreateFile(file_name);
-    if (INVALID_HANDLE_VALUE == hFile)
+    auto fileS = fio->_CreateFile(file_name, std::ios::binary | std::ios::in);
+    if (!fileS.is_open())
```

Comment:
we probably need logging here with level "verbose"

Reply:
As far as I can see - there is no such level:
https://github.com/gabime/spdlog/blob/v1.x/include/spdlog/common.h#L142

Reply:
Ok, I got it: Trace=verbose